apps/condo/domains/common/utils/importer.ts

```diff
@@ -172,98 +185,102 @@ export class Importer implements IImporter {
-
-                                        mutationErrors.forEach(mutationError => {
-                                            const mutationErrorMessages = get(mutationError, ['data', 'messages'], []) || []
-                                            mutationErrorMessages.forEach(message => {
-                                                const errorCodes = Object.keys(this.mutationErrorsToMessages)
-
-                                                errorCodes.forEach(code => {
-                                                    if (message.includes(code)) {
-                                                        normalizedRow.errors.push(this.mutationErrorsToMessages[code])
-                                                    }
-                                                })
-                                            })
-                                        })
-                                    } catch (error) {
-                                        console.debug('Error when formatting errors from "graphQLErrors"', {
-                                            row,
-                                            normalizedRow,
-                                            isValid,
-                                        })
-                                        console.error(error)
-                                        console.error(e)
-                                    }
-
-                                    if (this.failProcessingHandler) {
-                                        this.failProcessingHandler(normalizedRow)
-                                        return Promise.resolve()
-                                    }
-                                })
-                        } else {
-                            if (this.failProcessingHandler) {
-                                this.failProcessingHandler(normalizedRow)
-                                return Promise.resolve()
+
+        if (isNormalizedRow && isValidRow) {
+            try {
+                await sleep(this.sleepInterval)
+
+                await this.objectCreator(processedRow)
+
+                if (processedRow.shouldBeReported && this.failProcessingHandler) {
+                    this.failProcessingHandler(processedRow)
+                }
+                if (this.successProcessingHandler) {
+                    this.successProcessingHandler(row)
+                }
+            } catch (e) {
+                const mutationErrors = get(e, 'graphQLErrors', []) || []
+
+                if (!isArray(mutationErrors) || isEmpty(mutationErrors)) {
+                    console.error('Unexpected error in "objectCreator"!')
+                    console.error(e)
+                    processedRow.errors.push(this.errors.creation)
+                } else {
+                    for (const mutationError of mutationErrors) {
+                        const mutationErrorMessages = get(mutationError, ['data', 'messages'], []) || []
```

Comment:
it seems `get(mutationError, ['data', 'messages'], [])` is sufficient

Reply:
`get` replace only `undefined` value. If this value is `null`, then the replacement will not occur.
It seems that you can do without `|| []`, but who knows what the answer might come)